src/index.scss

```diff
@@ -289,3 +301,201 @@ section {
 #about {
 	background-color: white;
 }
+
+#schedule {
+	@extend .flex-container;
+	flex-direction: column;
+
+	max-height: none;
+	min-height: 100%;
+	height: auto;
+
+	.content {
+		@extend .flex-container;
+		flex-direction: column;
+
+		height: 100%;
+		width: 100%;
+
+		padding: 5vw;
+
+		@include mobile {
+			padding: 10vw;
+		}
+
+		.title {
+			align-self: start;
+			margin-bottom: 5vw;
+
+			@include mobile {
+				align-self: center;
+
+				position: absolute;
+				top: 5vw;
+			}
+		}
+
+		.schedule {
+			@extend .flex-container;
+
+			position: relative;
+
+			width: 60%;
+
+			border-radius: 1.5vw;
+			background-color: $light-blue;
+
+			@include mobile {
+				margin-top: 12.5vw;
+
+				flex-direction: column;
+				width: 85vw;
+				border-radius: 5vw;
+			}
+
+			.tape {
+				position: absolute;
+
+				height: 3vw;
+				width: 100%;
+
+				top: -1.75vw;
+
+				$bp-1: 17.5%; // tape start
+				$bp-2: 40%; // tape end
+
+				background: linear-gradient(to right, transparent $bp-1, 
+					$tape $bp-1, $tape $bp-2, 
+					transparent $bp-2,transparent 100-$bp-2,
+					$tape 100-$bp-2,$tape 100-$bp-1,
+					transparent 100-$bp-1);
+
+				@include mobile {
+					height: 8vw;
+					top: -5vw;
+
+					$bp-3: 40%;	// mobile tape width
+					background: linear-gradient(to right, 
+						transparent 50-$bp-3/2, $tape 50-$bp-3/2, 
+						$tape 50+$bp-3/2, transparent 50+$bp-3/2);
+				}
+			}
+
+			.dates {
+				@extend .flex-container;
+				flex-direction: column;
+
+				align-self: start;
+
+				flex: 20;
+
+				height: 100%;
+
+				margin-left: 2vw;
+
+				color: $schedule-date;
+
+				@include mobile {
+					flex-direction: row;
+					justify-content: space-between;
+
+					align-self: auto;
+
+					width: 80%;
+
+					margin: 2.5vw 0;					
+				}
+
+				.date {
+					flex: 100;
+					align-self: start;
+
+					@include mobile {
+						flex: none;
+					}
+					.day {
+						height: 8vw;
+
+						font-size: 7vw;
+						font-weight: 600;
+
+						@include mobile {
+							height: 16vw;
+							font-size: 14vw;
+						}
+					}
+					.weekday {
+						font-size: 1.6vw;
+						font-weight: 300;
+
+						@include mobile {
+							font-size: 3.2vw;
+						}
+					}
+				}
+			}
+
+			.events {
+				flex: 30;
+				margin: 3vw 3vw 3vw 0;
+
+				@include mobile {
+					width: 80%;
+					margin: 0 0 8vw 0;
+				}
+
+				.event {
+					@extend .flex-container;
+
+					align-items: flex-start;
+
+					flex-direction: column;
+
+					width: 100%;
+
+					padding: 1vw 0 0.75vw 2.25vw;
```

Comment:
Given that the parent block is `flex`, I don't think any of htese need to be `vw` or `vh`, same goes for margin and the gradient for the event color bit. 

Also, it doesn't appear that the `width` is doing anything because it's `flex` -- I tried toggling it in the web inspector, no difference

Reply:
Good catch on the width.

If I change the padding to use % over vw (e.g. 3.5% 0 2.5% 8%), it retains its proportions if I adjust the width (flex) which is not what I really want it to do.